types/fastify/index.d.ts

```diff
@@ -1,2 +1,3 @@
 export * from "./plugins";
+export * from "./auth";

```

Comment:
sort

Reply:
Plugins is a folder, that's why I placed it over there. Sort anyway?